src/virtual_ptdf_calculations.jl

```diff
@@ -148,21 +259,20 @@ function VirtualPTDF(
 end
 
 """
-Builds the PTDF matrix from a system. The return is a PTDF array indexed with the bus numbers.
-
-# Keyword arguments
-- `dist_slack::Vector{Float64}`: Vector of weights to be used as distributed slack bus.
-    The distributed slack vector has to be the same length as the number of buses
+Builds the Virtual PTDF matrix from a system. The return is a VirtualPTDF 
+struct with an empty cache.
 """
 function VirtualPTDF(
     sys::PSY.System; kwargs...)
     branches = get_ac_branches(sys)
-    nodes = get_buses(sys)
+    buses = get_buses(sys)
 
-    return VirtualPTDF(branches, nodes; kwargs...)
+    return VirtualPTDF(branches, buses; kwargs...)
 end
 
-# Overload Base functions
+"""
```

Comment:
There is a typo and also, check how the default is written 

```suggestion
"""
Determine whether the VirtualPTDF is empty (has no elements).
```